src/worker.ts

```diff
@@ -30,7 +30,7 @@ async function handleRequest (event: FetchEvent): Promise<Response> {
   try {
     request = await parseRequest(event.request)
   } catch (err) {
-    console.error(`Error while parsing request: ${err}`)
+    console.error(`Error while parsing request: ${String(err)}`)
```

Comment:
What's with these changes here? Its fine and should work ok but its definitely not any idomatic JS that I've seen

Reply:
The linting on the main branch specifically calls out "Invalid type "unknown" of template literal expression." on [that line](https://github.com/elementary/appcenter-payments-deb/actions/runs/13649414099/job/38154405582#step:5:13), and GitHub's AI said that was the fix. I agree it seems weird but I assume TypeScript is just being fussy.

Reply:
This is the part where we complain about TypeScript. Its not a great reason but its a reason.